blog/2024-04-05-translation-key-filtering.md

```diff
@@ -0,0 +1,17 @@
+---
+author: Joost Lekkerkerker
+authorURL: https://github.com/joostlek
+authorImageURL: https://avatars.githubusercontent.com/u/7083755?v=4
+title: "Changing the way translation keys are handled"
+---
+
+In Home Assistant Core 2024.4.2, we are changing the way translation keys are downloaded.
+
+Previously the script would download every translated string.
+With this change, we only download the strings that are verified.
+
+A verified string is a translated string, and it becomes unverified when the original string changes.
```

Comment:
I think this is better:
> All translations are verified when saved, but a translation becomes unverified id the default string is changed.

Reply:
```suggestion
All translations are verified when saved, but a translation becomes unverified if the source string is changed
```

Reply:
Since translation source is English:
> All translations are verified when saved, but a translation becomes unverified if the English string is changed.